packages/grafana-data/src/types/app.ts

```diff
@@ -126,6 +126,10 @@ export class AppPlugin<T extends KeyValue = KeyValue> extends GrafanaPlugin<AppP
  * @internal
  */
 export enum FeatureState {
+  /** @deprecated in favor of experimental */
   alpha = 'alpha',
+  /** @deprecated in favor of preview */
   beta = 'beta',
+  experimental = 'experimental',
+  preview = 'preview',
```

Comment:
What's the reasoning for sharing public and private preview?

Reply:
Not sure if I understand the question? We would for example use the Preview badge for features that are currently in `private` or `public` preview, to mark them.

Reply:
Based on what was communicated to me by @stevendungan and @zizzpudding we don't use `alpha` `beta` to mark features for a while now. The language has changed to `experimental` and `preview`.  

@mjseaman could you please provide guidance on this.

Reply:
Ah, sorry for not being clear. My reasoning is why don't we have two separate `Private preview` and `Public preview`?

Reply:
Oh! So we (obs logs) have been using 'Preview' for both, as it is not relevant to the user whether the preview is public or private. Also, 'Private/Public Preview' is quite lengthy.

Reply:
I guess so. If you look at the feature docs you can find the "real" type of preview. I am fine with okaying this, but if you want to you can verify with the product people.